pages/TheExplorationEditor.html

```diff
@@ -38,8 +38,8 @@
   </h3>
 
   <p>
-    A card consists of three parts: some 'content', an 'interaction' (or
-    question), and 'responses' that specify what Oppia should do next, based on
+    A card consists of three parts: some <em><strong>content</strong></em>, an <em><strong>interaction</strong></em> (or
```

Comment:
Per discussion in #1614, I thought we were going for just 'strong' here (no 'em')?